must-gather/gather

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+TIMESTAMP=$(date +%Y%m%d_%H%M%S)
+OUTPUT_DIR="${OUTPUT_DIR:-must-gather/${TIMESTAMP}}"
+
+DEFAULT_NS=$(oc get ns -o name | grep "oot" | cut -d"/" -f 2)
+NS="${NS:-$DEFAULT_NS}"
+
+mkdir -p "$OUTPUT_DIR"
+
+for resource in $(oc get all -n "$NS" --no-headers | awk '{print $1}'); do
+  type=$(echo "$resource" | cut -d'/' -f1 )
+  name=$(echo "$resource" | cut -d'/' -f2 )
+
+  echo "Inspecting ${resource}"
+  oc adm inspect -n "$NS" "$resource" --dest-dir="$OUTPUT_DIR/inspect"
+
+  echo "Collecting information for ${resource}"
+  describeFile="${OUTPUT_DIR}/${type}_${name}.yaml"
+  oc -n "$NS" describe "$resource" > "$describeFile"
+done
+
+echo "Collecting logs from 'deployment/oot-operator-controller-manager'"
+oc -n "$NS" logs "deployment/oot-operator-controller-manager" > "${OUTPUT_DIR}/oot-operator-controller-manager.log"
+
+echo "Inspecting modules.ooto.sigs.k8s.io in all the namespaces"
+oc adm inspect -A crd/modules.ooto.sigs.k8s.io  --dest-dir="$OUTPUT_DIR/inspect"
```

Comment:
This just inspects the `Module` CRD. We also want all `Modules` in all namespaces:
```suggestion
echo "Inspecting the modules.ooto.sigs.k8s.io CRD"
oc adm inspect crd/modules.ooto.sigs.k8s.io  --dest-dir="$OUTPUT_DIR/inspect"

echo "Inspecting all Modules in all namespaces"
oc adm inspect -A modules.ooto.sigs.k8s.io --dest-dir="$OUTPUT_DIR/inspect"
```